api/server/server_test.go

```diff
@@ -108,202 +87,3 @@ func getErrorResponse(t *testing.T, rec *httptest.ResponseRecorder) *models.Erro
 	}
 	return &err
 }
-
-func prepareDB(ctx context.Context, t *testing.T) (models.Datastore, models.LogStore, func()) {
-	os.Remove(tmpDatastoreTests)
-	uri, err := url.Parse("sqlite3://" + tmpDatastoreTests)
-	if err != nil {
-		t.Fatal(err)
-	}
-	ss, err := sql.New(ctx, uri)
-	if err != nil {
-		t.Fatalf("Error when creating datastore: %s", err)
-	}
-	logDB := logs.Wrap(ss)
-	ds := datastore.Wrap(ss)
-
-	return ds, logDB, func() {
-		os.Remove(tmpDatastoreTests)
-	}
-}
-
-func TestFullStack(t *testing.T) {
-	ctx := context.Background()
-	buf := setLogBuffer()
-	ds, logDB, close := prepareDB(ctx, t)
-	defer close()
-
-	rnr, rnrcancel := testRunner(t, ds)
-	defer rnrcancel()
-
-	srv := testServer(ds, &mqs.Mock{}, logDB, rnr, ServerTypeFull, LimitRequestBody(32256))
-
-	var bigbufa [32257]byte
-	rand.Read(bigbufa[:])
-	bigbuf := base64.StdEncoding.EncodeToString(bigbufa[:]) // this will be > bigbufa, but json compatible
-	toobigerr := errors.New("Content-Length too large for this server")
-	gatewayerr := errors.New("container exit code")
-
-	for _, test := range []struct {
-		name              string
-		method            string
-		path              string
-		body              string
-		expectedCode      int
-		expectedCacheSize int // TODO kill me
-		expectedError     error
-	}{
-		{"create my app", "POST", "/v1/apps", `{ "app": { "name": "myapp" } }`, http.StatusOK, 0, nil},
-		{"list apps", "GET", "/v1/apps", ``, http.StatusOK, 0, nil},
-		{"get app", "GET", "/v1/apps/myapp", ``, http.StatusOK, 0, nil},
-		// NOTE: cache is lazy, loads when a request comes in for the route, not when added
-		{"add myroute", "POST", "/v1/apps/myapp/routes", `{ "route": { "name": "myroute", "path": "/myroute", "image": "fnproject/fn-test-utils", "type": "sync" } }`, http.StatusOK, 0, nil},
-		{"add myroute2", "POST", "/v1/apps/myapp/routes", `{ "route": { "name": "myroute2", "path": "/myroute2", "image": "fnproject/fn-test-utils", "type": "sync"  } }`, http.StatusOK, 0, nil},
-		{"get myroute", "GET", "/v1/apps/myapp/routes/myroute", ``, http.StatusOK, 0, nil},
-		{"get myroute2", "GET", "/v1/apps/myapp/routes/myroute2", ``, http.StatusOK, 0, nil},
-		{"get all routes", "GET", "/v1/apps/myapp/routes", ``, http.StatusOK, 0, nil},
-		{"execute myroute", "POST", "/r/myapp/myroute", `{ "echoContent": "Teste" }`, http.StatusOK, 1, nil},
-
-		// fails
-		{"execute myroute2", "POST", "/r/myapp/myroute2", `{"sleepTime": 0, "isDebug": true, "isCrash": true}`, http.StatusBadGateway, 2, gatewayerr},
-		{"request body too large", "POST", "/r/myapp/myroute", bigbuf, http.StatusRequestEntityTooLarge, 0, toobigerr},
```

Comment:
this one doesn't seem captured by anything for invoke/triggers (no other instances of this status code in tests)

Reply:
just the 'request body too large' test

Reply:
Created in the fn invoke tests.